medical_lab/models/medical_lab_patient.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MedicalLabPatient(models.Model):
+    _name = 'medical.lab.patient'
+    _description = 'Medical Labs - Patient'
+    _inherits = {'medical.lab': 'lab_id'}
+
+    lab_id = fields.Many2one(
+        string='Lab',
+        comodel_name='medical.lab',
+        required=True,
+        ondelete='cascade',
+    )
+    state = fields.Selection(
```

Comment:
Shouldn't this state field be added to Lab Test (aka: medical.lab)? Then we could completely remove this model. Maybe it was introduced for some other reasons, then I think a comment in the code would be great.